scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/SubmitEditServlet.java

```diff
@@ -116,6 +117,8 @@ private void upload(HttpServletRequest request, HttpServletResponse response) {
                 varMap.put(name, val);
             } else if (key.toUpperCase().equals("START_AT")) {
                 startAt = val;
+            } else if (key.toUpperCase().equals("PLAN")) {
```

Comment:
I would recommend using "public boolean equalsIgnoreCase(String anotherString)" to compare Strings. It avoids copying data.

Reply:
Then I would also change it for the line right (old one) above to optimize both